src/composables/use-focus-on-blur.js

```diff
@@ -0,0 +1,68 @@
+import {
+  getDocument,
+  getActiveElement,
+  getNextActiveElementOnBlur,
+} from 'reakit-utils'
+import { ref, watch } from '@nuxtjs/composition-api'
+
+/**
+ * @typedef Props
+ * @property {import('./types').Ref<HTMLElement>} popoverRef
+ * @property {import('./types').ToRefs<import('../components/VPopover/VPopover.types').Props>} popoverPropsRefs
```

Comment:
WebStorm shows a warning that `VPopover.types` is not found.

Reply:
I don't think WebStorm is using TypeScript to interpret the JSDoc comments.